fvtest/rastest/subscriberForkTest.cpp

```diff
@@ -143,6 +143,12 @@ runTest(omrthread_entrypoint_t entrypoint)
 		}
 		postForkParent(vmthread, agent, &testVM, pipedata);
 	}
+
+	/* Clean up test output file */
```

Comment:
Are these files created by the agent? If so, they should be cleaned up by the agent.